### PR TITLE
Add: Redux implementation for authentication Remove: Context implemen…

### PR DESCRIPTION
**Pull request by aman007shrestha**
…tation for authentication

### CODE REVIEW

**Review by Panas18 — APPROVED** [review]
LGTM

**Review by SUSHILkhadka — APPROVED** [review]

File: src/components/LoginForm.tsx [hunk 1]
```diff
@@ -16,15 +17,16 @@ const initialCredential = {
 };
 
 export const LoginForm: React.FC = () => {
+  const dispatch = useDispatch();
   const navigate = useNavigate();
   const [credential, setCredential] = useState<Info>(initialCredential);
```
Comment (crazieeKinG) [inline src/components/LoginForm.tsx:22]: There is no use of this state as it is being maintained by redux. I think it is better to remove it as well as its setState in line 26.

**Review by crazieeKinG — APPROVED** [review]
Everything looks good.

File: src/components/LoginForm.tsx (hunk 1 above)
Comment (aman007shrestha) [reply src/components/LoginForm.tsx:22]: I will have the changes. Thank you

**Review by aayush-musyaju — APPROVED** [review]
Looks good to me

**Pull request merged**